core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -193,17 +193,17 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
     }
 
     // pkg private for testing
-    void execute(Terminal terminal, String pluginId, boolean isBatch, Environment env) throws Exception {
-        if (pluginId == null) {
-            throw new UserException(ExitCodes.USAGE, "plugin id is required");
+    void execute(Terminal terminal, String pluginName, boolean isBatch, Environment env) throws Exception {
```

Comment:
No, it should stay "id" in the message because plugins are installed by id (with the exception of some special plugins that can be installed by name only). Yet "name" is fine for removal because plugins are removed by name.